packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -121,13 +123,7 @@ module.exports = {
       // "url" loader embeds assets smaller than specified size as data URLs to avoid requests.
       // Otherwise, it acts like the "file" loader.
       {
-        exclude: [
-          /\.html$/,
-          /\.(js|jsx)$/,
-          /\.css$/,
-          /\.json$/,
-          /\.svg$/
-        ],
+        exclude: [/\.html$/, /\.(js|jsx)$/, /\.css$/, /\.json$/, /\.svg$/],
```

Comment:
The objective is indeed for the user to "accept" what the tool does and it'll make your life easier not to worry about every single small changes to your code.

However, since we are currently building the tool, please let me know about all those small things. I want to make sure that it looks the best it can.